scitt/check_operation_status.py

```diff
@@ -0,0 +1,46 @@
+""" Module for checking when a statement has been anchored in the append-only ledger """
+
+import json
+import subprocess
+import argparse
+import time
```

Comment:
Usually only import what is needed ising the from statement:

    from json import loads as json_loads
    from time import sleep as time_sleep

This keeps local dicts small and aids performance. Not a deal-breaker here but this is a nice habit to acquire in python code